CODIGO/General.bas

```diff
@@ -479,111 +479,125 @@ Private Sub CheckKeys()
     End If
 End Sub
 
-'TODO : Si bien nunca estuvo alla­, el mapa es algo independiente o a lo sumo dependiente del engine, no va aca¡!!!
 Sub SwitchMap(ByVal Map As Integer)
-'**************************************************************
-'Formato de mapas optimizado para reducir el espacio que ocupan.
-'Diseñado y creado por Juan Martin Sotuyo Dodero (Maraxus) (juansotuyo@hotmail.com)
-'**************************************************************
-    Dim Y As Long
-    Dim X As Long
-    Dim tempint As Integer
-    Dim ByFlags As Byte
-    Dim handle As Integer
-    Dim CharIndex As Integer
-    Dim obj       As Integer
-    
+    '**********************************************************************************
+    'Dise�ado y creado por Juan Mart�n Sotuyo Dodero (Maraxus) (juansotuyo@hotmail.com)
+    '**********************************************************************************
+    
+    '**********************************************************************************
+    'Formato de mapas optimizado para reducir el espacio que ocupan.
+    'Nueva carga de mapas desde la memoria (clsByteBuffer)
+    '[ https://www.gs-zone.org/temas/carga-de-mapas-desde-la-memoria-cliente.91444/ ]
+    '**********************************************************************************
+
+    Dim Y        As Long
+    Dim X        As Long
+    Dim tempint  As Integer
```

Comment:
This var is never being in use, it can be deleted.

Reply:
```
    'Load arrays
    For Y = YMinMapSize To YMaxMapSize
        For X = XMinMapSize To XMaxMapSize
```
They´re being used there.

Reply:
```
    'Load arrays
    For Y = YMinMapSize To YMaxMapSize
        For X = XMinMapSize To XMaxMapSize
```
They´re being used there